Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'danger-dangermattic', '~> 1.0'
 gem 'fastlane', '~> 2.216'
 gem 'fastlane-plugin-appcenter', '~> 2.1'
 gem 'fastlane-plugin-sentry', '~> 1.14'
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 12.2'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 12.4'
```

Comment:
ℹ️  This update is in particular to be able to benefit from [this fix on `extract_release_notes_for_version`(https://github.com/wordpress-mobile/release-toolkit/pull/623)